### PR TITLE
Add istio support docs

### PR DESCRIPTION
**Pull request by hmahmood**
### What does this PR do?
Adds a blurb about istio support in the agent.

### Motivation
Newly released istio support

### Preview

https://docs-staging.datadoghq.com/hmahmood-add-istio-support/content/en/network_performance_monitoring/installation.md

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/network_performance_monitoring/installation.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ The following **provisioning systems** are supported:
 - Chef 12.7+: See the [Datadog Chef recipe][6]
 - Ansible 2.6+: See the [Datadog Ansible role][7]
 
+**Istio** versions v1.6.4 and above are supported by [Datadog Agent v7.24.1+][1].
```
Comment (shlomoartzi) [inline content/en/network_performance_monitoring/installation.md:47]: copy looks good! can we move this above the line on Cilium?

Comment (hmahmood) [reply content/en/network_performance_monitoring/installation.md:47]: Sure

**Review by leeavital — APPROVED** [review]

File: content/en/network_performance_monitoring/installation.md [hunk 2]
```diff
@@ -32,6 +32,8 @@ Supported **platforms** include:
 
 There is an exemption to the 4.4.0+ kernel requirement for [CentOS/RHEL 7.6+][3]. The [DNS Resolution][4] feature is not supported on CentOS/RHEL 7.6.
 
+**Istio** versions v1.6.4 and above are supported by [Datadog Agent v7.24.1+][1].
```
Comment (ruthnaebeck) [inline content/en/network_performance_monitoring/installation.md:35]:
```suggestion
**Istio** v1.6.4+ is supported by [Datadog Agent v7.24.1+][1].
```

**Review by ruthnaebeck — APPROVED** [review]
👍

**Pull request merged**